npm/angular/README.md

```diff
@@ -6,7 +6,7 @@ Mount Angular components in the open source [Cypress.io](https://www.cypress.io/
 
 ## Requirements
 
-- Angular 17.2.0+ (Cypress 13 and under supports Angular 13 - 16)
+- Angular 18.0.0+ (Cypress 13 and under supports Angular 13 - 16)
```

Comment:
@AtofStryker This note is now a bit weird with it mentioning no way to run Angular 17

Reply:
I have these updated now to include the npm package versions for which versions of angular are supported in which package version